docs/guides/README.md

```diff
@@ -0,0 +1,3 @@
+# Guides
+
+We've created few guides that are intended to answer common questions or demonstrate testing strategies in more detail. If there are guides that you would like to see in the future, check out [the `roca` project open issues](https://github.com/hulu/roca/issues) to see if it's already being tracked, and create a new issue if needed.
```

Comment:
Possible typo: "We've created few guides"

technically true since one is very few 😛

Reply:
Hey hey hey with this PR, we'll now have _two_ whole guides 😉 

but also yeah that's a typo